styles.css

```diff
@@ -21,6 +21,10 @@ body {
 	z-index: 10;
 }
 
+#noscript h1 {
+	font-size: 1.8em;
```

Comment:
Your additions made the message span two lines, which didn't exactly look great. This reduces the font size so it all fits on one line.